worlds/tunic/__init__.py

```diff
@@ -373,7 +385,8 @@ def fill_slot_data(self) -> Dict[str, Any]:
             "Hexagon Quest Holy Cross": self.ability_unlocks["Pages 42-43 (Holy Cross)"],
             "Hexagon Quest Icebolt": self.ability_unlocks["Pages 52-53 (Icebolt)"],
             "Hexagon Quest Goal": self.options.hexagon_goal.value,
-            "Entrance Rando": self.tunic_portal_pairs
+            "Entrance Rando": self.tunic_portal_pairs,
+            "disable_local_spoiler": int(tunic_settings["disable_local_spoiler"])
```

Comment:
Suggestion: Use trailing commas in collection initializers, so that the changeset doesn't always need to touch the previous line